apps/organizations/templates/organizations/cms/organization.html

```diff
@@ -0,0 +1,11 @@
+{% extends "richie/fullwidth.html" %}
+{% load cms_tags %}
+{% block content %}
+<h1>{{ current_page.get_title }}</h1>
+{% with current_page.organizationpage.organization as organization %}
+    {% if organization.logo %}
+    <img src="{{ organization.logo.url }}">
+    {% endif %}
+{% endwith %}
+{% placeholder "maincontent" %}
+{% endblock content %}
```

Comment:
We should discuss templates indentation.

Reply:
I modified it.